controllers/user.server.controller.js

```diff
@@ -29,7 +29,7 @@ exports.userCreate = function(req, res) {
         'username': reqBody.username && reqBody.username.length > 2 ? reqBody.username : false,
         'displayName': reqBody.displayName && reqBody.displayName.length > 2 ? reqBody.displayName : generateDisplayName(),
         'email': reqBody.email && emailValidator.validate(reqBody.email) ? reqBody.email : false,
-        'hashedPassword': reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
+        'password': reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
```

Comment:
```suggestion
        'plaintextPassword': reqBody.password && reqBody.password.length > 0 ? reqBody.password : false
```
If we are going to use a more readable name then why not use plaintextPassword to more clearly define that it is not hashed yet

Reply:
Sounds good to me, done.